src/reducers/summaryReducer.js

```diff
@@ -0,0 +1,26 @@
+import {
+  SUMMARY_FETCHING,
+  SUMMARY_RECEIVED,
+} from '../actions/actionTypes'
+
+const initialState = {
+  loading: false,
+}
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case SUMMARY_FETCHING:
+      return {
+        ...state,
+        loading: true,
+      }
+    case SUMMARY_RECEIVED:
+      return {
+        ...state,
+        ...action.summaries,
```

Comment:
Expanding any of the summary objects onto the state object, so that we can store at least the national summary and the state summary in question